src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -0,0 +1,93 @@
+# Basic matching pipeline that takes in 
+
+#Input:
+# 1. MERGED_ONTOLOGY = tmp/merged.owl
+# 2. SSSOM_CONFIG = metadata/mondo.sssom.config.yml
+# 3. OUTPUT_SSSOM = mapping/mondo-sources-all-lexical.sssom.tsv
+
+# I would try some basic things first:
+
+# Use synonymiser
+# Use oak.mapping() pipeline
+
+import logging
+from pathlib import Path
+from oaklib.resource import OntologyResource
+from oaklib.implementations.sqldb.sql_implementation import SqlImplementation
+from oaklib.utilities.lexical.lexical_indexer import (
+    create_lexical_index,
+    lexical_index_to_sssom,
+    load_mapping_rules,
+    save_lexical_index
+)
+from sssom.writers import write_table
+import sys
+from pathlib import Path
+import yaml
+import click
+
+SRC = Path(__file__).resolve().parents[1]
+ONTOLOGY_DIR = SRC / "ontology"
+OUT_INDEX_DB = ONTOLOGY_DIR / "tmp/merged.db.lexical.yaml"
```

Comment:
Is this way to construct paths platform independent? I.w. will it work on windows?

Reply:
Yes, here's a [blog](https://realpython.com/python-pathlib/) that states it.

Reply:
Thank you :)